src/frontend/src/containers/AllStatus.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { AppState } from '../redux/store';
+import Spinner from '../components/Spinner';
+import NoSearchResults from '../components/NoSearchResults';
+
+type Props = {
+  loading: boolean;
+};
+
+class AllStatus extends React.Component<Props> {
+  render() {
+    return (
+      <section>
+        {this.props.loading ? <Spinner /> : <NoSearchResults />}
```

Comment:
This all looks great! One small thing is that the app will render "No search results found." before a search has been executed. I don't think that is super important at this point in time though!

Reply:
I was thinking about the same thing.  If we wanted tp workaround that we would have to add a bit more logic which is not a big deal.  "No Search Results Found" however is not a false statement so I left it as is unless we feel that this is a change worth doing.